persistence/src/main/java/rocks/metaldetector/persistence/domain/spotify/SpotifyAuthorizationEntity.java

```diff
@@ -0,0 +1,35 @@
+package rocks.metaldetector.persistence.domain.spotify;
+
+import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import rocks.metaldetector.persistence.domain.BaseEntity;
+import rocks.metaldetector.persistence.domain.user.UserEntity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.OneToOne;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PACKAGE) // for hibernate and model mapper
```

Comment:
to avoid IntelliJ warning

```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED) // for hibernate and model mapper
```